internal/logger/protohandler.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Ultraviolet
+// SPDX-License-Identifier: Apache-2.0
+package logger
+
+import (
+	"context"
+	"io"
+	"log/slog"
+
+	"github.com/ultravioletrs/cocos/pkg/manager"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+var _ slog.Handler = (*handler)(nil)
+
+type handler struct {
+	opts slog.HandlerOptions
+	w    io.Writer
+}
+
+func NewProtoHandler(w io.Writer, opts *slog.HandlerOptions) slog.Handler {
+	if opts == nil {
+		opts = &slog.HandlerOptions{}
+	}
+	return &handler{
+		opts: *opts,
+		w:    w,
+	}
+}
+
+// Enabled implements slog.Handler.
```

Comment:
Remove full stop at the end of the comments

Reply:
@drasko  top-level comments should end in a period according to Golang https://go.dev/wiki/CodeReviewComments